src/nvhttp.cpp

```diff
@@ -1096,7 +1150,47 @@ namespace nvhttp {
    */
   void
   erase_all_clients() {
-    map_id_client.clear();
+    client_t client;
+    client.uniqueID = "0123456789ABCDEF"s;
```

Comment:
Same question... why hard coded?

Reply:
See previous response. It might be unnecessary.

Reply:
Removed 👍 (https://github.com/LizardByte/Sunshine/pull/2042/commits/5ff80ada0b93dd8527c0c7e20e42e9f75065b0f3)